### DISCUSSION TITLE
Unable to change Username Using Firefox

### DISCUSSION THREAD

**Discussion by nigelnquande** [community/community#109544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I would like to change my username/URL slug on Github. However, when I followed the instructions to do so on my profile (by clicking the <kbd>Change Username</kbd> button on the appropriate page), I was not shown the warning dialog/modal. However, the following appeared in the console/debug window when using "Inspect Element" from the context menu:

````JS
Uncaught ReferenceError: HTMLDialogElement is not defined
    tM dialog_helper.js:23
    connectedCallback dialog_helper.js:84
    38168 dialog_helper.js:118
````

For reference, I'm using Mozilla Firefox 88.0 for Ubuntu on Ubuntu 16. (Yeah, I know, it's outdated, but so is my laptop.)

**Comment by arcticsoftxyz** [comment]
You may have an outdated browser!  
The error `Uncaught ReferenceError: HTMLDialogElement is not defined` tells us exactly why you can't see the dialog: Firefox 88 (and many browsers for a while) didn't fully support the` <dialog> `HTML element.  
GitHub likely uses this element for the username change confirmation.

You should **update your Firefox browser**. Or, you can **use another updated browser**.
